extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -49,8 +64,11 @@ public static function onBeforePageDisplay( \OutputPage $out/*, \Skin $skin*/ )
 				'wgRelatedVideoId' => $relatedVideoData['videoId'],
 			] );
 		}
+	}
 
-		return true;
+	private static function addJWPlayerAssets( \OutputPage $out, $featuredVideoData ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$out". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$featuredVideoData". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)